package.json

```diff
@@ -2,9 +2,37 @@
   "name": "reactjs-course-2",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "web": "live-server dist",
+    "build": "webpack --mode=development",
+    "build:prod": "webpack --mode=production",
+    "lint": "eslint . --ext .js,.tsx",
+    "lint:fix": "eslint --fix --ext .js,.jsx ."
+  },
+  "dependencies": {
+    "jest": "^26.6.3",
+    "lodash": "^4.17.20",
```

Comment:
Do you really need lodash package in your app. Bear in mind it is very heavyweight, and most of the methods you can easily produce with ES6 array methods.

Reply:
Idk, that was in the "getting started" selection of webpack documentation :D 
I thought without Jest it wont work, but it's fine. I think in next commit I'll delete it.